### PR TITLE
adds Jerry Zhang to the team

### PR DESCRIPTION
**Pull request by y4h006969**
I'm not sure why this hasn't happened yet... [Jerry Zhang](https://github.com/jerryzhang222) is an invaluable member of the Screwdriver team.  He has done some very important work, as evidenced [here](https://github.com/screwdriver-cd/screwdriver/pull/437), [here](https://github.com/screwdriver-cd/scm-bitbucket/pull/33), [here](https://github.com/screwdriver-cd/scm-bitbucket/pull/32), [here](https://github.com/screwdriver-cd-test/quickstart-generic/pull/5), [here](https://github.com/screwdriver-cd/launcher/pull/101), [here](https://github.com/screwdriver-cd/data-schema/pull/105) and [here](https://github.com/screwdriver-cd/screwdriver/pull/437).
This PR adds Jerry Zhang to the team:

#### Before
screen shot 2017-01-30 at 10 17 05 pm

#### After
screen shot 2017-01-30 at 10 16 20 pm

cc @tkyi @jerryzhang222 @stjohnjohnson for review & merge! 😄

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by y4h006969** [comment]
I just agreed to a CLA 👍 :shipit:

File: _config.yml [hunk 1]
```diff
@@ -69,6 +69,11 @@ people:
   position: Digital Plumber
   github: https://github.com/jer
 
+- name: Jerry Zhang
+  pic: jerry
```
Comment (stjohnjohnson) [inline _config.yml]: The picture you uploaded is named `jayz` so this needs to be changed.

**Comment by y4h006969** [comment]
@tkyi @jerryzhang222 any thoughts on this?  Looking to :ship: :it: today 😸

**Comment by y4h006969** [comment]
Thanks @tkyi 
@stjohnjohnson can we verify this is :100: and get it merged?

**Comment by y4h006969** [comment]
giphy

**Pull request merged**